05/mobile.css

```diff
@@ -0,0 +1,203 @@
+@media (min-width: 10px) and (max-width: 800px){
+body{
+    max-width: 800px;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+.all{
+    max-width: 800px;
+    max-height: 100vh;
+    
+}
+.header{
+    display: grid;
+    grid-template-columns: minmax(800px);
+    grid-template-rows: minmax(200px);
+    height: 99%;
+    border: solid;
+    border-color: green;  
+}
+
+.header__row{
+    display:grid;  
+    grid-template-columns: minmax(3, 150px 300px 300px);
+    grid-template-rows: minmax(200px);
+    border: solid;
+    border-color: rgb(31, 19, 207);
+    height: 80px;
+    grid-template-areas: 
+    "A B C D";
+}
+
+.header__element{
+    display: grid;
+    grid-area: B;
+    place-self: center;
+    width: 180px;
+    height: 40px;
+    background-color: yellow;
+}
+
+
+nav{
+    display: grid;
```

Comment:
Zwróć uwagę, że wszędzie jest wpisany "grid" czy faktycznie jest tutaj tyle tych elementów doustiania, aby korzystać wszędzie z `grid`? W zasadzie to jedynie `header__row` ma "dzeici" do ustawienia. W reszcie przy rządków wystarczy pewnie domyślny `display: block`.

Wracając do `nav`, zrezygnowałbym ze wszystkich właściwości. Dla `nav` dodałbym: `display: inline-block`, aby móc określić jago "wymiary" - wtedy wartości `height` i `width` zaczną działać. Takie będzie tego efekt:

![image](https://github.com/devmentor-pl/practice-html-and-css-rwd/assets/55095257/afe86628-8ef6-413f-9ea0-e8c71d68cdc3)

PS. Teraz ten element należy ułożyć przy pomocy grida :P Czyli względem rodzica tj. `.header__row`